collector/recon.go

```diff
@@ -37,23 +37,38 @@ type ReconCollector struct {
 }
 
 // NewReconCollector creates a new ReconCollector.
-func NewReconCollector(pathToExecutable string, isTest bool) *ReconCollector {
+func NewReconCollector(pathToExecutable string, isTest, diskUsage, driveAudit, md5, quarantined, replication, unmounted, updaterSweepTime bool) *ReconCollector {
```

Comment:
Having a ton of arguments of the same type is not considered good style because it's too easy to mix up the order when calling the function. Also, function calls like `NewReconCollector(path, true, true, true, true, true, true, ...)` are not very readable. It would be nicer to define an options struct to allow for what is essentially named arguments:

```go
collector := NewReconCollector(path, ReconCollectorOpts {
  IsTest: true,
  WithDiskUsage: true,
  WithDriveAudit: true,
  ...
})
```

Reply:
Ahh, yes. I remember this from when the go-bits backoff review. I need to get rid of this bad habit. PR updated.